TypeCobol.Test/Parser/Programs/Cobol2002/TypedefInCopy/typedef-strong.rdzMix-EI.txt

```diff
@@ -1,9 +1,9 @@
-Line 1[27,33] <27, Error, Syntax> - Syntax error : Custom EI rule : Only TYPEDEF STRICT is allowed.
        IDENTIFICATION DIVISION.
        PROGRAM-ID. MainProgram.
        data division.
        working-storage section.
 Line 5[27,33] <27, Error, Syntax> - Syntax error : Custom EI rule : Only TYPEDEF STRICT is allowed.
+Line 5[27,33] <27, Error, Syntax> - Syntax error : Custom EI rule : Only TYPEDEF STRICT is allowed.
```

Comment:
Ideally, we should avoid duplicate errors. I guess that what is happening here is that we get one error from the copy and one from the usage of the type. So to avoid that, the checker should only raise one error but not both.

Reply:
done: adding to  the message "Syntax error : Custom EI rule : Only TYPDEF STRICT is allowed" "in definition of type <name_of_the_typedef>.".
https://github.com/TypeCobolTeam/TypeCobol/pull/1602/commits/c75e93cbbbe150f7b2561c1696660261a82adfbe

Reply:
In fact, here we get 2 errors, because there are 2 typedef NOT strict.
To improve readability, I propose to add the name of the COPY inside the message.
Something like:
Line 5[27,33] <27, Error, Syntax> - Syntax error in XXXX.cpy : Custom EI rule : Only TYPEDEF STRICT is allowed.

Reply:
done; retrieve if exists copy name and add to messages in method Cobol2002Checker.CheckTypedef
example message "Syntax error : Custom EI rule : Only TYPEDEF STRICT is allowed in definition of type TypedefWeak1 in 'TypedefWeak' copy"
https://github.com/TypeCobolTeam/TypeCobol/pull/1602/commits/b3858d1f5a5d9ac4d5ab15c99198b4222fbf0e3d